tests/kafkatest/tests/streams/streams_optimized_test.py

```diff
@@ -65,30 +66,41 @@ def test_upgrade_optimized_topology(self):
 
         processors = [processor1, processor2, processor3]
 
-        # produce records continually during the test
+        self.logger.info("produce records continually during the test")
```

Comment:
Switched comments to logs so that I could tell what phase of the test we were in while reading the logs.